src/GdbConnection.h

```diff
@@ -688,6 +689,10 @@ class GdbConnection {
   // true when "no-ack mode" enabled, in which we don't have
   // to send ack packets back to gdb.  This is a huge perf win.
   bool no_ack;
+  // contains signals which should be passed directly to the
+  // debuggee without gdb being informed, speeding up
+  // reverse execution
+  std::unordered_set<int> pass_signals;
```

Comment:
Please add a note that these are *gdb* signal numbers not native signal numbers.